src/main/java/org/reactome/release/cosmicupdate/Main.java

```diff
@@ -0,0 +1,337 @@
+		validateConfigValue(Main.COSMICMutationTrackingURL, "URL for COSMIC Mutation Tracking file cannot be null/empty! Please set a value for urlToMutationTrackingFile in the application's properties file");
+		validateConfigValue(Main.COSMICFusionExportURL, "URL for COSMIC Fusion Export file cannot be null/empty! Please set a value for urlToFusionExportFile in the application's properties file");
+		
+		COSMICFileRetriever mutantExportRetriever = new COSMICFileRetriever();
+		COSMICFileRetriever mutationTrackingRetriever = new COSMICFileRetriever();
+		COSMICFileRetriever fusionExportRetriever = new COSMICFileRetriever();
+		
+		try
+		{
+			mutantExportRetriever.setDataURL(new URI(Main.COSMICMutantExportURL));
+			mutationTrackingRetriever.setDataURL(new URI(Main.COSMICMutationTrackingURL));
+			fusionExportRetriever.setDataURL(new URI(Main.COSMICFusionExportURL));
+			
+			final String mutantExportDestination = "./CosmicMutantExport.tsv.gz";
+			final String mutationTrackingDestination = "./CosmicMutationTracking.tsv.gz";
+			final String fusionExportDestination = "./CosmicFusionExport.tsv.gz";
+			
+			mutantExportRetriever.setFetchDestination(mutantExportDestination);
+			mutationTrackingRetriever.setFetchDestination(mutationTrackingDestination);
+			fusionExportRetriever.setFetchDestination(fusionExportDestination);
+			
+			logDownloadMessage(mutantExportRetriever, mutantExportDestination);
+			this.executeDownload(mutantExportRetriever);
+			
+			logDownloadMessage(mutationTrackingRetriever, mutationTrackingDestination);
+			this.executeDownload(mutationTrackingRetriever);
+			
+			logDownloadMessage(fusionExportRetriever, fusionExportDestination);
+			this.executeDownload(fusionExportRetriever);
+		}
+		catch (URISyntaxException e)
+		{
+			e.printStackTrace();
+			throw e;
+		}
+		catch (Exception e)
+		{
+			logger.error("Error occurred while trying to download a file! " + e.getMessage(), e);
+			// throw the exception further up the stack - can't continue if file downloads are failing.
+			throw e;
+		}
+	}
+
+	private void logDownloadMessage(COSMICFileRetriever retriever, final String destination)
+	{
+		logger.info("Downloading {} to {}", retriever.getDataURL(), destination);
+	}
+	
+	/**
+	 * Executes a single download.
+	 * @param retriever - the file retriever to run.
+	 * @throws Exception
+	 */
+	private void executeDownload(COSMICFileRetriever retriever) throws Exception
+	{
+		retriever.setMaxAge(this.fileAge);
+		retriever.setUserName(Main.COSMICUsername);
+		retriever.setPassword(Main.COSMICPassword);
+		retriever.fetchData();
+	}
+	
+	/**
+	 * Updates the identifiers that need updating.
+	 * @param adaptor
+	 * @param updates
+	 * @throws Exception 
+	 */
+	private static void updateIdentifiers(MySQLAdaptor adaptor, Map<String, List<COSMICIdentifierUpdater>> updates) throws Exception
+	{
+		for (List<COSMICIdentifierUpdater> updater : updates.values())
+		{
+			updater.forEach(u -> {
```

Comment:
If `updater` is renamed `updaterList`, it would help to rename `u` as `updater` since here it is iterating over each one individually.